timed/tracking/views.py

```diff
@@ -173,7 +173,7 @@ def update(self, request, *args, **kwargs):
         }
         if fields and request.user != instance.user:
             tasks.notify_user_changed_report(instance, fields, request.user)
-        if fields.get("rejected"):
+        if fields.get("rejected") and request.user != instance.user:
             tasks.notify_user_rejected_report(instance, request.user)
```

Comment:
If notifications are never sent in case where the user does stuff to own reports, that could be made more explicit by writing all related to notification below a `if request.user != instance.user`